src/gui/QvisColorTableWindow.C

```diff
@@ -510,7 +511,14 @@ QvisColorTableWindow::CreateNode(DataNode *parentNode)
         // Save the current color table.
         std::string ct(currentColorTable.toStdString());
         node->AddNode(new DataNode("currentColorTable", ct));
-        node->AddNode(new DataNode("tagList", tagList));
+        stringVector tagNames;
+        std::vector<bool> activeTags;
```

Comment:
Wonder if it makes sense to use `boolVector` here too for consistency?

Reply:
yes it does, thanks!